docs/griffe_custom_decorator_labels.py

```diff
@@ -4,8 +4,7 @@
 """
 
 # noinspection PyProtectedMember
-from griffe.agents.visitor import stdlib_decorators
-from griffe.extensions import Extension
+from griffe import Extension, stdlib_decorators
```

Comment:
The changes in this file seem to have broken the documentation builds.

Reply:
It looks like griffe just deprecated importing submodules like this in version 0.48.0. What's the best way to handle this?

Reply:
It looks like this is caused by a slight difference between the `pyproject.toml` file and the `docs/requirements.txt` file. My recommendation is to add `griffe = "^0.47.0"` to the `[tool.poetry.group.docs.dependencies]` section in the `pyproject.toml` file and then run the `poetry update --sync` command to update your local environment. The requirements file used to build docs is locked to griffe v0.47.0, but your local environment is allowing v0.48.0 due to the difference between the two files. Then these changes in this file can be reverted.

Reply:
Let me know if you need any more assistance with this.

Reply:
Your suggestion was good, I've worked through this. I'm working on testing now.

Reply:
Great! You should just be able to expand the existing MDO2kB simulated device file to test the new functionality. Let me know if you have any questions.